package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-pretty",
-  "version": "7.1.0",
+  "version": "7.2.0",
```

Comment:
We will bump this ourselves, please revert.

Reply:
That's great! I'll revert this